bom/pom.xml

```diff
@@ -536,6 +545,18 @@
             </plugin>
          </plugins>
       </pluginManagement>
+      <plugins>
+         <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <extensions>true</extensions>
+            <configuration>
+               <nexusUrl>${jboss.releases.nexus.url}</nexusUrl>
+               <serverId>${jboss.releases.repo.id}</serverId>
+               <autoReleaseAfterClose>true</autoReleaseAfterClose>
```

Comment:
the infinispan-bom should be user-consumable, while infinispan-parent is our actual parent, where we define plugin configs, lifecycle, etc. If polluting the bom is the only way (I hope not) then everything should be relegated in a release-specific profile (e.g. distribution)